notebooks/official/model_monitoring/batch_prediction_model_monitoring.ipynb

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
<p>all !cmd should have space between ! and cmd.</p><p> </p><p>pip -&gt; pip3</p><p> </p><p>add -q</p><p> </p><p>do not pin 'google-cloud-aiplatform'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>fixed</p>